lib/linux.js

```diff
@@ -41,18 +41,23 @@ var LinuxBuilder = {
       linux.archName = 'x86_64';
     }
 
-    var tmpFolder = path.join( os.tmpDir(), linux.executable );
+    var outFilename = linux.executable + '-' +
```

Comment:
Not a fan of standard either... :) cool! Ja but we can surely tweak the rules if anything is really annoying you.

As long it s consistent I don't mind that much. :)